src/components/MultiHopTrade/components/TradeInput/TradeInput.tsx

```diff
@@ -140,12 +143,29 @@ export const TradeInput = memo(() => {
   const manualReceiveAddressIsValidating = useAppSelector(selectManualReceiveAddressIsValidating)
   const sellAmountCryptoPrecision = useAppSelector(selectSellAmountCryptoPrecision)
   const slippageDecimal = useAppSelector(selectTradeSlippagePercentageDecimal)
+  const activeQuoteErrors = useAppSelector(selectActiveQuoteErrors)
+  const quoteRequestErrors = useAppSelector(selectTradeQuoteRequestErrors)
+  const isUnsafeQuote = useAppSelector(selectIsUnsafeActiveQuote)
 
   const hasUserEnteredAmount = useMemo(
     () => bnOrZero(sellAmountCryptoPrecision).gt(0),
     [sellAmountCryptoPrecision],
   )
-  const activeQuoteStatus = useActiveQuoteStatus()
+  const quoteStatusTranslation = useMemo(() => {
+    const quoteRequestError = quoteRequestErrors[0]
+    const tradeQuoteError = activeQuoteErrors?.[0]
+    switch (true) {
+      case !!quoteRequestError:
+        return getQuoteRequestErrorTranslation(quoteRequestError)
+      case !!tradeQuoteError:
+        // this should never occur because users shouldn't be able to select and errored quote
```

Comment:
something something grep OCD
```suggestion
        // this should never occur because users shouldn't be able to select an errored quote
```

Reply:
8c08173ce32972327469c01af0ae86087d5fb237